<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>networkProfilerDialogBase</class>
 <widget class="QDialog" name="networkProfilerDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Network Profiler</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblNetwork">
       <property name="text">
        <string>Layer:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="ctlLayer"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lblReachID">
       <property name="text">
        <string>Start reach ID:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ctlReachInt"/>
     </item>
     <item>
      <widget class="QPushButton" name="cmdGetReachFromMap">
       <property name="text">
        <string>From Selection</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lblWarning">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="lblFields">
       <property name="text">
        <string>Fields:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>430</width>
          <height>242</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="txtCSVOutput"/>
     </item>
     <item>
      <widget class="QLabel" name="lblCSV">
       <property name="text">
        <string>CSV output:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmdBrowseCSV">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="cmdButtons">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cmdButtons</sender>
   <signal>accepted()</signal>
   <receiver>networkProfilerDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdButtons</sender>
   <signal>accepted()</signal>
   <receiver>networkProfilerDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
